src/reducers/RequestTypeReducer.js

```diff
@@ -0,0 +1,30 @@
+import * as Constants from '../constants/indexConstants.js'
+
+const initialState = {
```

Comment:
please add tests when adding new reducers
follow the existing reducer tests as an example

Reply:
done